tools/manifest/tests/test_sourcefile.py

```diff
@@ -761,7 +761,7 @@ def test_xhtml_with_entity(ext):
 
 
 def test_no_parse():
-    s = create("foo/bar.xml", "\uFFFF".encode("utf-8"))
+    s = create("foo/bar.xml", "\uFFFF".encode())
```

Comment:
This is an unrelated change. See https://github.com/web-platform-tests/wpt/pull/29301/commits/0de54c04a1649c520cb08acd8394973cce1ce932 for what I reverted.

Is there a way to prevent pyupgrade from making this change in the future?

Reply:
Yes, I should have caught that - I've removed it from the changes